coremltools/test/neural_network/test_numpy_nn_layers.py

```diff
@@ -3382,7 +3382,14 @@ def test_reshape_like_cpu(self, cpu_only=True):
                 inputs = {'data': data, 'tensor': tensor}
                 expected = {'output': np.reshape(data, target_shape)}
 
-                self._test_model(builder.spec, inputs, expected, useCPUOnly=cpu_only)
+                #Make sure GPU doesn't have too much batch
+                max_batch_size_gpu = 400
+                if any([ dim > max_batch_size_gpu for dim in target_shape]):
```

Comment:
Can we just lower the `dim` in `target_shape`?

Reply:
It's unnecessary to have this logic here and waste testing resources as well.

Reply:
Reduce the `target_shape` size. I think it's generated randomly by `np.random.randint()`. We can make it smaller.

Reply:
Done :)

Reply:
LGTM!